### PR TITLE
GMO Sign (Certified Connector)

### PR DESCRIPTION
**Pull request by GMO-GSHD-APIGW**
179186684-e9c23411-86a2-47a7-9967-37df45bd8d45
179186767-1a0dac62-86a4-4581-9337-061c3b7226c8
179186811-c5fe35c6-c096-46e4-87c2-af13402039f6

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by v-EgorKozhadei** [comment]
Hi @GMO-GSHD-APIGW,

Thank you for your submission! 

I noticed that you named [your previous PR](https://github.com/microsoft/PowerPlatformConnectors/pull/1810) as if it were an independent publisher connector. And this one looks like it's a verified publisher connector.

Are you the owner of this service or do you work at the company that owns this API?
We need to make sure that everything is correct, that's why I'm asking about it.

I briefly looked through the artifacts you've submitted, all strings should be in English language. You can refer to [English language strings guideline](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings) for more information.

Please let us know if you have any questions.

File: certified-connectors/GMO Sign/apiProperties.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+	"properties": {
```
Comment (v-EgorKozhadei) [inline certified-connectors/GMO Sign/apiProperties.json:2]: @GMO-GSHD-APIGW please fix the indentation in apiProperties.json file.

File: certified-connectors/GMO Sign/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+			  "tabIndex": 3,
+			  "clearText": false,
+			  "required": "true"
+			}
+		  }
+		}
+	  },
+	  "iconBrandColor": "#005bac",
+	  "capabilities": [],
+	  "publisher": "GMO GlobalSign Holdings K.K."
```
Comment (v-EgorKozhadei) [inline certified-connectors/GMO Sign/apiProperties.json:33]: @GMO-GSHD-APIGW stackOwner property is missing. Please add it. You can refer to [apiProperties template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).

File: certified-connectors/GMO Sign/Readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Title
```
Comment (v-EgorKozhadei) [inline certified-connectors/GMO Sign/Readme.md:1]: @GMO-GSHD-APIGW please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1).

File: certified-connectors/GMO Sign/apiProperties.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+	"properties": {
+	  "connectionParameters": {
+		"username": {
+		  "type": "securestring",
+		  "uiDefinition": {
+			"displayName": "顧客ID",
```
Comment (v-EgorKozhadei) [inline certified-connectors/GMO Sign/apiProperties.json:7]: @GMO-GSHD-APIGW please use English strings for all connector files. You can refer to [English language strings guidelines](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1) and to [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance).

File: certified-connectors/GMO Sign/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,907 @@
...
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.gmogshd.com/privacy"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Content and Files;Productivity"
+    }
+  ],
+  "host": "y4omo61inf.execute-api.ap-northeast-1.amazonaws.com",
```
Comment (v-EgorKozhadei) [inline certified-connectors/GMO Sign/apiDefinition.swagger.json:30]: @GMO-GSHD-APIGW can you please confirm that this host URL is for production environment? Using test or staging environment URLs is not allowed.

File: certified-connectors/GMO Sign/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,907 @@
...
+                },
+                "urlToPublishTo": {
+                  "type": "string",
+                  "x-ms-summary": "Webhook用URL",
+                  "x-ms-visibility": "internal",
+                  "x-ms-notification-url": true
+                },
+                "envelopeEvents": {
+                  "type": "string",
+                  "x-ms-summary": "トリガーを引く封筒状態",
```
Comment (v-EgorKozhadei) [inline certified-connectors/GMO Sign/apiDefinition.swagger.json:86]: @GMO-GSHD-APIGW please use English strings for all connector files. You can refer to [English language strings guidelines](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1) and to [connector string guidance](https://docs.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance).

**Review by v-EgorKozhadei — COMMENTED** [review]
Hello @GMO-GSHD-APIGW!

Please check comments after the review.
You can also remove the icon file from the PR, it will be required in next steps.

Let us know if you have any questions.

**Comment by v-EgorKozhadei** [comment]
Hello @GMO-GSHD-APIGW, 

I hope you are doing well. 

I am following up with you to see how things are going with preparing your connector. 

Please let us know if you have any questions. 

Thank you.

**Comment by v-EgorKozhadei** [comment]
Hello @GMO-GSHD-APIGW,

Thank for updating string values!
I resolved two comments for English string values from previous review, others are not resolved.
Please resolve the remaining comments after updating the files with fix.
Also the icon is not required in the PR, could you please remove it? You'll be able to submit an icon during the next step of certification.

Let us know if you have any questions.

**Comment by GMO-GSHD-APIGW** [comment]
Hello @v-EgorKozhadei

Our "GMO Sign" service is available in English, Japanese, Vietnamese, Filipino, and other languages.
But it is primarily intended for use in Japan.
This is because, in general, electronic signature contract services require functional support based on the laws and regulations of each country.
Therefore, we would like to include Japanese in the readme.md and UI.
Is it prohibited to do so at all?

**Review by sriyen-msft — APPROVED** [review]

**Comment by v-EgorKozhadei** [comment]
Hello @GMO-GSHD-APIGW, 

I hope you are doing well. 

Your connector will be localized in different languages because connectors are localized as part of Power Automate localization.

Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file. 

After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6. 

Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.  

We expect all tests to be completed within 1-2 weeks. 

If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions. 

Please let me know if you have any questions or run into any problems while creating or updating your submission. 

Thank you very much for working with us.

**Pull request merged**